integr8ly/installation
# Fuse dashboard imports

## Additional Information

JIRA: https://issues.jboss.org/browse/INTLY-602

*What*
Install the monitoring ‘addon’ CustomResources during integreatly installation.
https://github.com/syndesisio/syndesis/tree/22beae32f32fe42f3bb696f85ae5ef5acf938267/install/addons

*Why*
This will provide a number of Alerts & Dashboards that the Integreatly monitoring stack will detect and import.
These can be used to debug & diagnose Fuse issues



## Verification Steps
1. From the fork ``https://github.com/davidkirwan/intly-installation.git`` checkout branch ``fuse_dashboards`` on the openshift bastion/master instance.
2. Create the inventory with the master and key etc configured to point at the openshift master instance.
3. ``ansible-playbook -i inventories/hosts playbooks/install.yml -e eval_self_signed_certs=false`` Should run to completion successfully.
4. The port ``9090`` should also be exposed on the ``prometheus-service`` service.
5. application-monitoring-operator should be ``0.0.13`` in the ``middleware-monitoring`` namespace.
6. grafana operator should show importing of fuse dashboards successfully in its logs.
7. Visiting Grafana dashboard should show the fuse dashboards successfully (eventually).
8. ``ansible-playbook -i inventories/hosts playbooks/uninstall.yml -e eval_self_signed_certs=false`` should run to completion successfully.


## Is an upgrade task required and are there additional steps needed to test this?
